### DISCUSSION TITLE
希望后续能优化设备操作界面

### DISCUSSION THREAD

**Discussion by yyxida** [XiaoMi/ha_xiaomi_home#18] [Ideas]
很多设备的操作面板都是粗暴的直接把接口变成按钮扔在上面，感觉就像是内部用开发文档批量转换的一样，操作体验十分的差（当然我知道今天才是发布 第一天，这样要求有点太高了）。
比如踢脚线的控制面板，HA有现成的加热器面板，（隔壁miot就是用的这个），希望能够早日改进吧，至少把现在常用的电暖器加湿器空调啥的改一下，夏天的设备可以晚点适配。
说个好的方面，能支持本地中枢是最大的优点，让我剁手了犹豫许久的be3600pro。

**Comment by yyxida** [comment]
![BBWP`28EG)8D`TOR H~6(BU](https://github.com/user-attachments/assets/3ac8da39-1d79-45da-a14d-dbaa67543661)
![@)UUY%@MC)2NTZJX$RBMA 0](https://github.com/user-attachments/assets/b861b70b-a7ec-4d73-a99a-62bf3b5a65fa)

左为本项目，右为第三方miot插件

**Comment by topsworld** [comment]
感谢建议，下个版本适配电暖器
